examples/webgl_lensflares.html

```diff
@@ -112,7 +112,7 @@
 
 				// renderer
 
-				renderer = new THREE.WebGLRenderer( { antialias: true, alpha: true } );
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
```

Comment:
I remember this was set on purpose my @mrdoob for performance reasons. I'll try to find the related issue/PR.

Reply:
Unfortunately, I find only the commit but not the discussion: 1937e58e8eedff74f2a8c2d3274a56c098cddc38

Reply:
Here is the discussion 😇 https://github.com/mrdoob/three.js/issues/13170#issuecomment-365142026

Reply:
Hmm, we can always revert if needed.